products/BellHybrid/apps/application-bell-main/widgets/DuoHBox.cpp

```diff
@@ -0,0 +1,43 @@
+// Copyright (c) 2017-2021, Mudita Sp. z.o.o. All rights reserved.
```

Comment:
```suggestion
// Copyright (c) 2017-2022, Mudita Sp. z.o.o. All rights reserved.
```

Reply:
this should be fixed automatically, @paweljonskim please check your commit hooks